app/views/schools/participants/show.html.erb

```diff
@@ -178,21 +178,55 @@
       <% end %>
     </dl>
 
-    <% if @ects.present? %>
-      <h3 class="govuk-heading-m"><%= @induction_record.participant_full_name.split(" ").first %> is mentoring these
-        ECTs</h3>
-
-      <%= govuk_summary_list(classes: ["govuk-!-margin-bottom-7"]) do |list| %>
-        <% @ects.each do |ect| %>
-          <% list.with_row do |row| %>
-            <% row.with_key(text: ect.full_name,
-                       classes: ["govuk-!-font-weight-regular"]) %>
-            <% if @mentors_added &&
-                  ect.participant_profile.policy_class.new(current_user, ect.participant_profile).update_mentor? %>
-              <% row.with_action(text: "Change",
-                            visually_hidden_text: "mentor",
-                            href: school_participant_edit_mentor_path(participant_id: ect.participant_profile_id,
-                                                                      from_mentor: @profile.id)) %>
+    <% if @profile.mentor? %>
+      <h3 class="govuk-heading-m">Mentoring</h3>
+
+      <%= govuk_summary_list(classes: ["govuk-!-margin-bottom-7"], actions: false) do |slist| %>
+        <% slist.with_row do |row| %>
+          <% row.with_key do %>
+            Currently<br>mentoring
+          <% end %>
+          <% row.with_value do %>
+            <% if @ects.present? %>
+              <%= govuk_summary_list(classes: ["govuk-!-margin-bottom-7"], borders: false) do |list| %>
```

Comment:
Can we take a look at the nested summary list here or was it built this way for a specific reason? In the prototype the content inside the main summary list is structured with a regular unordered list

It means that both lists have a margin-bottom-7 on, which throws out the spacing on the bottom of that row.

Reply:
So it is fine if I just remove the css class from the nested summary-list?

Reply:
That'd look ok visually, but accessibility-wise maybe it should be a `<ul>` rather than a nested `<dl>` here? 🤔 

In the prototype I used a `<ul>` but then a bunch of `<div>`s with grid column classes within each `<li>` to get the 'Change' link to appear right-aligned: https://github.com/DFE-Digital/manage-training-for-early-career-teachers-prototype/blob/main/app/views/mentors/show.html#L95-L114

Possibly either is fine though, I'm not an accessibility expert on this.

Reply:
From a dev perspective it is much easier and cleaner to use the nested summary-list, I would say.

I have removed the extra margin so if we can leave it like that it would be great, otherwise I would have to add explicitly that buch of divs and css classes you used in your prototype making it all kind of dirty and messy.

But if you guys strongly think we need to change it to an unsorted list instead, i will do it.

Reply:
Nested summary list replaced with <ul>